src/servers/trigger.ts

```diff
@@ -181,7 +181,7 @@ export const onchainBlockEventHandler = async (height: number) => {
 export const invoiceUpdateEventHandler = async (invoice: GetInvoiceResult) => {
```

Comment:
I think an important test on this PR is to check (manually, on devnet or testnet, most tests doesn't rely on triggered  event), that the event is being trigger when the invoice is been held.

also see what happened if trigger is being restarted after an invoice has been created but not paid.

https://github.com/lightningnetwork/lnd/issues/3120

I went through the code here but not clear to me if there is a different of behavior with held invoice or not (this is what I remember when I first did this PR some months ago)
https://github.com/alexbosworth/lightning/blob/master/lnd_methods/invoices/subscribe_to_invoices.js